### ISSUE TITLE
Search and export list

### ISSUE THREAD

**Issue by camillemonchicourt** [makinacorpus/Geotrek#1582]
Je fais une recherche libre dans le module ITINERAIRES par exemple. 
La liste des résultats et la carte se limite aux itinéraires correspondant à la recherche.
Cependant quand j'exporte en SHP, celui-ci contient tous les itinéraires, et pas seulement correspondant à la recherche libre.

**Comment by LePetitTim** [comment]
It seems to be fix. Isn't it ?

**Comment by camillemonchicourt** [comment]
No, the export dont use the search from the bottom of lists. It only uses filters.

![search](https://user-images.githubusercontent.com/4418840/42282056-a0c172f8-7fa5-11e8-9ec1-232be0e6f2e9.png)

And it seems to be the same in all modules.

**Comment by babastienne** [comment]
Toujours le cas en 2.100.1. Pour que l'export de fichier soit filtré il faut utiliser les filtres issus du module dédié, mais en effet le filtre par recherche textuelle de datatable n'affecte pas les résultats exportés.
